packages/react-component-library/src/components/AutocompleteE/AutocompleteE.tsx

```diff
@@ -11,23 +11,58 @@ import {
 } from '../SelectBase'
 import { NoResults } from './NoResults'
 import { useHighlightedIndex } from './hooks/useHighlightedIndex'
-import { useInput } from './hooks/useInput'
-import { useItems } from './hooks/useItems'
+import { useAutocomplete } from './hooks/useAutocomplete'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface AutocompleteEProps extends SelectBaseProps {}
 
+function onBlur(resetHighlightedIndex: () => void) {
```

Comment:
I find the name of these a little bit confusing, as they create event handlers rather than handle the event directly.

So would suggest something like e.g. `createBlurHandler` (or, alternatively, turn them into hooks in which case `useCallback` could be used as well).